### ISSUE TITLE
Taito Chase H.Q. + S.C.I. (Japan)

### ISSUE THREAD

**Issue by goukizero** [tpunix/SAROO#136]
The problem is that each game has only one audio track. They are multi-track, contain several voices (radio) that must be played at different times and only play the first time.

In the fenrir it stays played on a loop

**Comment by Mikeylovesretro** [comment]
I gave this one a go and did not have any issues.

**Comment by tzmwx** [comment]
Version 0.5 software currently does not support audio single track multi segment.

**Comment by tzmwx** [comment]
> I gave this one a go and did not have any issues.

How to detect it? Enter the game, select "Option", select "MC", only 00 has sound, and 01-40 has no sound.

**Comment by goukizero** [comment]
as far as it works badly in all odes

**Comment by tpunix** [comment]
It comes: multi index support!
[mcuapp.zip](https://github.com/user-attachments/files/15766105/mcuapp.zip)

**Comment by tzmwx** [comment]
> The problem is that each game has only one audio track. They are multi-track, contain several voices (radio) that must be played at different times and only play the first time.
> 
> In the fenrir it stays played on a loop

Note:
 After testing, fenrir is in the wrong operating mode. It should play a segment, not the entire song. 
The new software of TPU runs fine. 

You can download the test, pay attention to the testing method, enter OPTION, enter MC, and select 00-40

**Comment by Retro-Nerd** [comment]
Yes, it works fine for. But this update mcuapp update from yesterday has a regression for Thunder Force Gold Pack 1. Both games freezes at the title screen after a few seconds. Old v0.5 mcuapp works fine.

**Comment by goukizero** [comment]
with this mcuapp update thunder force v it does not have cd sound, for example

**Comment by tpunix** [comment]
There is a fixed version.Please test it again:
[mcuapp.zip](https://github.com/user-attachments/files/15805709/mcuapp.zip)

**Comment by goukizero** [comment]
New archive:
Chase hq OK
 Thunder Force Gold Pack 1  OK
 Thunder Force Gold Pack 2  OK
 Thunder Force v                   OK
Magical drop3 (M_6037A96=0009)   OK

Everything seems correct now, as far as I know only Hebereke's Popoitto (black screen) fails.
Thank you very much for your time and your work, I will continue testing to see if there are any other errors.

**Comment by odiaboeeu** [comment]
Could someone test the virtua racing if now the songs during the race are playing correctly?

**Comment by goukizero** [comment]
Forgive me mate, but VR from arcade, md, 32x, ss, ps2....
After 10/15 seconds the music ends, it's like that because Sega programmed it that way
There is a "hack" for saturn with audio tracks of:
Flag to flag dc
Ff355 dc
Hang on ss
Manxtt ss
Daytona 2 arcade
Formula one megacd
------------------------------------
Grand chaser ss
Evan K blue lightning

**Comment by odiaboeeu** [comment]
It always plays music when he passes a checkpoint, but in the saroo he is only playing once at the start of the race

**Comment by nullpointer-br** [comment]
> Could someone test the virtua racing if now the songs during the race are playing correctly?

No music at checkpoints... this problem still need to be fixed.

**Comment by goukizero** [comment]
I'm right with you on that, I thought the problem was the short time the tracks played.
But it's true that it only sounds the first time

**Comment by goukizero** [comment]
Hebereke's Popoitto runs fine

Black screen for mi, any configuration in the txt for it?

**Comment by goukizero** [comment]
ok I'm trying Japanese (black screen)
You will be testing the pal that works perfectly

**Comment by lagious** [comment]
KOF95 freezes at the SEGA logo after 240610 version (At 240608 it  runs fine)

**Comment by Mikeylovesretro** [comment]
Can confirm that KOF95 freezes on Sega logo  - 240611
